src/build.py

```diff
@@ -1401,12 +1401,23 @@ def TestEmtestAsm2Wasm():
       warn_only=False)
 
 
+def TestWasmSimd():
+  script = os.path.join(SCRIPT_DIR, 'test_wasm_simd.py')
+  clang = Executable(os.path.join(INSTALL_BIN, 'clang'))
+  include = os.path.join(EMSCRIPTEN_SRC_DIR, 'system', 'include')
+  try:
+    proc.check_call([script, clang, include])
```

Comment:
Instead of `proc.check_call`, we could `import test_wasm_simd` and call `test_wasm_simd.test_wasm_simd(clang, include)` directly